pyfakefs/fake_pathlib.py

```diff
@@ -383,6 +385,7 @@ class _FakeWindowsFlavour(_FakeFlavour):
         | {"COM%d" % i for i in range(1, 10)}
         | {"LPT%d" % i for i in range(1, 10)}
     )
+    pathmod = ntpath
```

Comment:
This class attribute was present but seemingly unused before https://github.com/python/cpython/pull/95450